ddtrace/contrib/internal/tornado/application.py

```diff
@@ -34,21 +36,25 @@ def tracer_config(__init__, app, args, kwargs):
     service = settings["default_service"]
 
     # extract extra settings
+    # TODO: Remove `FILTERS` from supported settings
     trace_processors = settings.get("settings", {}).get("FILTERS")
 
-    # the tracer must use the right Context propagation and wrap executor;
-    # this action is done twice because the patch() method uses the
-    # global tracer while here we can have a different instance (even if
-    # this is not usual).
-    tracer._configure(
+    tracer.configure(
```

Comment:
Should this not be removed as well? From the PR description it sounds like all `tracer.configure` calls are being removed.

Reply:
We're just removing `tracer._configure` for now. `tracer.configure` is still part of the public API. We will fully remove it in v4.0